.goreleaser.yaml

```diff
@@ -128,6 +128,27 @@ docker_manifests:
       - "garethgeorge/backrest:{{ .Tag }}-scratch-arm64"
       - "garethgeorge/backrest:{{ .Tag }}-scratch-armv6"
 
+brews:
+  - name: backrest
+    homepage: https://github.com/garethgeorge/backrest
+
+    url_template: "https://github.com/garethgeorge/backrest/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
+
+    commit_author:
+      name: goreleaserbot
+      email: bot@goreleaser.com
+
+    repository:
+      owner: garethgeorge
+      name: homebrew-backrest-tap
+      branch: main
+      token: "{{ .Env.GH_BREW_TOKEN }}"
+
+    service: |
```

Comment:
This is _really_ neat. Didn't know that homebrew had the capability to manage service execution, this is strictly better than the `install.sh` creating a launchd agent directly from XML.